### PR TITLE
Day two grind

### PR DESCRIPTION
**Pull request by Abaii**

### CODE REVIEW

File: src/Components/Button/Button.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import { NavButton } from './Button.styles.js';
+import { StyledButton } from './Button.styles.js';
 
 export const Button = (props) => {
-    const {label,type} = props;
+    const {type,functionality,label} = props;
     return (
-        <NavButton type={type}>{label}</NavButton>
+        <StyledButton type={type} onClick={()=>functionality(label)}>{label}</StyledButton>
```
Comment (Abaii) [inline src/Components/Button/Button.js:7]: depdency on label, make button more generalised. Rename `functionality` to `onClick`

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,39 @@
...
-import { Button } from './Components/Button/Button.js'
+import { Container, Col, Row } from 'react-bootstrap';
+import { Navbar } from './Components/Navbar/Navbar.js';
+import { HomePage } from './Components/HomePage/HomePage.js';
+import { Banner } from './Components/Banner/Banner.js';
 
 function App() {
+  const [ page, updatePage ] = useState('Home');
+
+  const changePage = (directTo) => {
```
Comment (Abaii) [inline src/App.js:11]: rename argument to `page` more readble

File: src/App.js [hunk 3]
```diff
@@ -1,11 +1,39 @@
...
+    if (page === 'Details') {return <div>Details page placeholder</div>};
+    if (page === 'Log in') {return <div>Log in page placeholder</div>};
+  };
+
   return (
     <div>
-      <Button label={'Home'} type={'primary'}></Button>
+      <Container>
+        <Row>
+          <Col md='auto'><Banner/></Col>
```
Comment (Abaii) [inline src/App.js:27]: you can define width of based on number of columns it takes in the row. Auto will evenly space them which is why you need an empty column

File: src/Components/Button/Button.js [hunk 4]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import { NavButton } from './Button.styles.js';
+import { StyledButton } from './Button.styles.js';
 
 export const Button = (props) => {
-    const {label,type} = props;
+    const {type,functionality,label} = props;
```
Comment (Abaii) [inline src/Components/Button/Button.js:5]: add spacing in your destructuring `const { type, functionality, label } = props;`

File: src/Components/Button/Button.js (hunk 1 above)
Comment (Jack-Writer) [reply src/Components/Button/Button.js:7]: Still need to do this

**Pull request merged**